### DISCUSSION TITLE
Swagger API page is empty

### DISCUSSION THREAD

**Discussion by victarary** [apache/superset#18311] [Q&A / Help]
After deploying superset in docker, the swagger page(/swagger/v1) displayed nothing except the superset menu header.  
The configuration `FAB_API_SWAGGER_UI` has been set to `True` in `superset_config.py`.

### Expected results

All superset APIs should be listed.

### Actual results

An empty page.

#### How to reproduce the bug

Go to https://host:port/swagger/v1
An empty page was displayed

### Environment

- superset version: `0.999.0`

**Comment by dpgaspar** [comment]
@victarary 

docker-compose on this repo will not use your `superset_config.py`

**Comment by victarary** [comment]
@dpgaspar 
Actually, I have put superset_config.py under pythonpath in container superset_app and restarted docker-compose.
Everything has taken effect except `FAB_API_SWAGGER_UI`.

**Comment by last-Programmer** [comment]
I am also facing this issue. Not sure how to fix this. thanks
